viewing_party/party.py

```diff
@@ -0,0 +1,168 @@
+def create_movie(title, genre, rating):
+    if not title:
+        return None
+    if not genre:
+        return None
+    if not rating:
+        return None
```

Comment:
```suggestion
  if not title or not genre or not rating:
        return None
```

You can combine conditions into one. Both ways achieve the same goal of returning None if any of the parameters are falsy, just packs it into the least amount of lines without sacrificing readability TOO much.